src/scripts/clipperUI/components/previewViewer/previewViewerAugmentationHeader.tsx

```diff
@@ -52,23 +52,25 @@ class PreviewViewerAugmentationHeaderClass extends PreviewViewerHeaderComponentB
 	private getSerifGroup(): ControlGroup {
 		return {
 			id: Constants.Ids.serifControl,
+			role: "radiogroup",
+			isAriaSet: true,
 			innerElements: [
 				<button
 					aria-label={Localization.getLocalizedString("WebClipper.Accessibility.ScreenReader.ChangeFontToSansSerif")}
-					aria-pressed={!this.props.serif ? "true" : "false"}
+					aria-checked={!this.props.serif + ""}
 					id={Constants.Ids.sansSerif}
-					{...this.enableInvoke(this.props.changeFontFamily, 101, false) }
+					{...this.enableInvoke(this.props.changeFontFamily, !this.props.serif ? Constants.ConditionalTabIndices.oneHundredAndOne : undefined, false, undefined, Constants.AriaSet.serifGroupSet) }
```

Comment:
Sorry, I meant a constant in this class or method. It's overkill to have this declared in Constants.ts as every file will then have access to it.